codegen-luau/runtime/numeric.lua

```diff
@@ -27,6 +27,7 @@ local table_freeze = table.freeze
 local from_u32, into_u32, from_u64, into_u64
 local num_add, num_subtract, num_multiply, num_divide_unsigned, num_negate
 local num_is_negative, num_is_zero, num_is_equal, num_is_less_unsigned, num_is_greater_unsigned
+local bit_not_rs
```

Comment:
Should be renamed to `num_not`, and moved to the end of line 28 to follow the pattern.